kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -621,6 +638,20 @@ public static ConfigDef newConfigDef() {
           Width.SHORT,
           "Behavior for null-valued records"
       );
+
+      configDef.define(
```

Comment:
We should add a configTest to verify recommenders, validators are working appropriately. 👍